File: 04 Kontrollstrukturen/Pascal/Aufgabe2.pas
Program Aufgabe2;

{$codepage utf8}

Var Alter: Integer;

Begin
    Write('Bitte gib das Alter ein: ');
    Read(Alter);

    If Alter <= 0 Then
        WriteLn('Es wurde ein ungültiges Alter angegeben!')
    Else If Alter <= 12 Then
        WriteLn('Der Kunde ist ein Kind.')
    Else If Alter >= 18 Then
        WriteLn('Der Kunde ist erwachsen.')
    Else
        WriteLn('Der Kunde ist ein Jugendlicher.');
End.

File: 04 Kontrollstrukturen/Go/Aufgabe2.go
package main

import "fmt"

var alter int

func main() {
	fmt.Print("Bitte gib das Alter ein: ")
	fmt.Scan(&alter)

	if alter <= 0 {
		fmt.Println("Es wurde ein ungültiges Alter angegeben!")
	} else if alter <= 12 {
		fmt.Println("Der Kunde ist ein Kind.")
	} else if alter <= 18 {
		fmt.Println("Der Kunde ist ein Jugendlicher.")
	} else {
		fmt.Println("Der Kunde ist erwachsen.")
	}
}
